.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+Thank you for your contribution!
+
+If this is the first time you are contributing to lorri, please take a look at:
+
+https://github.com/target/lorri/CONTRIBUTING.md
+-->
+
+Closes #ISSUE. <!-- Please replace ISSUE by the issue number this pull request addresses. -->
```

Comment:
Let’s put the comment on the line above so you can jump to the end of the line and delete word.

Reply:
Done